src/components/LoginPage/colors.js

```diff
@@ -0,0 +1,9 @@
+export default {
+    white: '#fff',
+    black: '#000',
+    aqua: '#96ffff',
+    rainfull: '#131C3E',
+    denim: '#0E1736',
+    pearl: '#E8E9F8',
+    blue: '#4A51F5'
+}
```

Comment:
Please always add a newline at end of file (configure your editor to do so by default).

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I have many similar issues. At the beginning I had Prettier... Could I return it?

Reply:
Sure, if it helps.

I remember when you had prettier you pushed a lot of irrelevant changes (changing code style in places which you shouldn't have touched). It isn't a good thing too... Because I'll have to review all those changes. So if you use it, please use it only for the code you write/change, but not for the existing code.

Reply:
For now I have not so many lines, so I did it by hand. Done in commit [d4c8246](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/d4c8246bda8ed820f84057146ad71487778e5f4f)